workflow/controller/controller.go

```diff
@@ -306,6 +313,8 @@ func (wfc *WorkflowController) startLeading(ctx context.Context, logCtx *log.Ent
 	for i := 0; i < podWorkers; i++ {
 		go wait.Until(wfc.podWorker, time.Second, ctx.Done())
 	}
+
+	go wait.UntilWithContext(ctx, wfc.syncAllConfigMaps, 10*time.Second)
```

Comment:
every 10s - I would expect this to cause too many allocs and result in OOMKilled - why do we need this?

Reply:
This is to prepare future work, e.g. processing of individual configmaps and GC when condition is met. Any suggestions? I can change this to less frequent, e.g. 1 minute and make it configurable via env var.

Reply:
I am going to remove this part in this PR and add later when needed.